README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
```

Comment:
This looks good; I like the ordered lists for game play instruction and that you made sure to credit the creator!